resources/templates/netCore/Dockerfile.template

```diff
@@ -4,6 +4,18 @@ WORKDIR /app
 EXPOSE {{ . }}
 {{/each}}
 
+{{#if (eq platform '.NET: ASP.NET Core')}}
+{{#if ports}}
+ENV ASPNETCORE_URLS=http://+:{{ ports.[0] }}
+{{/if}}
+{{/if}}
+
+{{#unless (isRootPort ports)}}
+# Switching to a non-root user, please refer to https://aka.ms/vscode-docker-python-user-rights

```

Comment:
TODO: fix the link

Reply:
Made one, https://aka.ms/vscode-docker-dotnet-configure-containers. Currently points to the Python doc.